src/state/megaStore.tsx

```diff
@@ -139,11 +136,15 @@ export const Provider: ParentComponent = (props) => {
                 await initializeWasm();
                 setState({ load_stage: "setup" });
 
+                const settings = await getSettings();
+
                 const mutinyWallet = await setupMutinyWallet(
                     settings,
                     password
                 );
 
+                setState({ settings: settings });
```

Comment:
isn't this redundant, we are getting the settings right before

Reply:
that's just setting it on the global store so other components can access settings (instead of looking it up from localstorage or re-running getsettings), I'll add a comment